### ISSUE TITLE
only vagrant user availabe

### ISSUE THREAD

**Issue by GiuseppeFiorentino** [p4lang/tutorials#460]
after the installation, with vagrant up ecc. only the user vagrant is available.

how can I switch to P4 user?
Or can I use also the vagrant ones?

**Comment by jafingerhut** [comment]
If things are working well, you should boot up to a screen that lets you pick whether to log in with whatever user you want that exists, either p4 or vagrant.  After logging in, you can choose to log out, and be taken back to that same screen that lets you log in to either account again.

If you are not seeing that, then it sounds like something went wrong when you built the VM.  If you would like to try a precompiled VM, there are several images built using VirtualBox available here: https://github.com/jafingerhut/p4-guide/blob/master/bin/README-install-troubleshooting.md

**Comment by joshuamhtsang** [comment]
@GiuseppeFiorentino Have you solved this?

I ran into a similar problem but it transpired that I didn't let the Vagrantfile's scripts run to completion.  To fix, I did:

1. Open VirtualBox Manager and 'Delete' the VM (usually called 'P4 Release Version XXXX-XX-XX).
2. Redo the `vagrant up` process.
3. Wait until vagrant process returns command line to you.  Do not close/restart the VM before this step completes, took me about 15 mins.

Upon completion you will see the VM present a GUI:
![image](https://user-images.githubusercontent.com/7430706/166444554-5a592d29-9c36-4432-8130-aacdc4aeb378.png)
